docs/manage/troubleshooting_server_faq.md

```diff
@@ -158,6 +158,45 @@ They will only work once the intake pipeline has run and moved the data into the
 Trips are converted from draft -> processed when the intake pipeline has been run successfully on them.
 - Instructions for running the intake pipeline for a single user (in debug mode): https://github.com/e-mission/e-mission-server#quick-start
 - Instructions for running the intake pipeline as part of a cronjob: https://github.com/e-mission/e-mission-server/wiki/Deploying-your-own-server-to-production#the-analysis-pipeline
+Actually there's no instructions on how to setup an actual cronjob, so here's my little tutorial.
+Since we are using source to set the environment for the intake script to run in, we create a little bash script by the name cron_intake.sh:
+```
+#!/bin/bash
+source activate emission
+./e-mission-py.bash bin/intake_multiprocess.py 4
+```
+make sure it's executable:
+```
+chmod a+x cron_intake.sh
+```
+Since we've couldn't figure out how to get the cronjob working inside the docker container, we are going to create the cronjob outside of the container on the host machine. As a first test, we can run this script from outside the container:
```

Comment:
@sosias there is now an example of how to run the cronjob inside of docker in a multi-container setup.
https://github.com/e-mission/e-mission-docker/tree/master/examples/em-server-multi-tier-cronjob

I suspect this is much easier than running the cronjob on the host machine, and it is also more consistent with serverless execution where you may not have access to the host machine. @xubowenhaoren verified that it works, and was going to improve the related documentation.

Maybe you can replace this with a link to that example, and improve that documentation as needed.

Reply:
Nice resource! I didn't saw that. I added the link to the e-mission-docker examples & made some change in text. Maybe is good to leave them both, so we have guides to run the cronjob inside and outside the container, if somebody need them.